sdk/samples/iot/provisioning/src/generate_certificate.sh

```diff
@@ -2,11 +2,15 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # SPDX-License-Identifier: MIT
 
+set -o errexit # Exit if command failed.
+set -o nounset # Exit if variable not set.
+set -o pipefail # Exit if pipe failed.
+
 openssl ecparam -out device_ec_key.pem -name prime256v1 -genkey
 openssl req -new -days 365 -nodes -x509 -key device_ec_key.pem -out device_ec_cert.pem -config x509_config.cfg -subj "/CN=paho-sample-device1"
-openssl x509 -text -noout -fingerprint -in device_ec_cert.pem
+openssl x509 -noout -text -in device_ec_cert.pem
 
-rm device_cert_store.pem
+rm -f device_cert_store.pem
```

Comment:
Out of curiosity, what motivated adding the `-f` option?

Reply:
it was in the same file in the other directory and I copied to make them uniform.  I do not know enough about the details of these commands right now to say.  scripts will be more investigated later.